export/Dockerfile

```diff
@@ -1,3 +1,21 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.15.14 AS crane-builder
+RUN git clone https://github.com/konveyor/crane $APP_ROOT/src/github.com/konveyor/crane
+WORKDIR $APP_ROOT/src/github.com/konveyor/crane
+## Workaround go 1.15 builder. Remove this after go-toolset 1.16 is available
+RUN sed -i 's/\"os\"/"os"\n\t"io\/ioutil"/g' cmd/apply/apply.go
+RUN sed -i 's/os\.ReadFile/ioutil.ReadFile/g' cmd/apply/apply.go
+RUN sed -i 's/go 1.16/go 1.15/g' go.mod
+## End Workaround go 1.15 builder.
+RUN go build -o crane main.go
+
+FROM registry.access.redhat.com/ubi8/go-toolset:1.15.14 AS plugin-builder
+RUN mkdir -p $APP_ROOT/src/github.com/konveyor
```

Comment:
Doing this before the add below to prevent permissions problems. You could do `ADD --chown ...` below, but buildah (downstream) does not respect it. https://bugzilla.redhat.com/show_bug.cgi?id=1937087